### PR TITLE
Added lab 2

### PR DESCRIPTION
**Pull request by andxzrey**
Added lab 2

### CODE REVIEW

File: labs/andrey_gluschuk/lab_2/task_1/5.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    1. Дано ціле число. Якщо воно є додатним, то відняти від нього 8; в іншому разі не змінювати його.
+    Вивести отримане число.
+'''
+
+x = int(input("Введіть ціле число: "))
+
+if x > 0:
+    print(x - 8)
+else:
+    print(x)
```
Comment (sen-den) [inline labs/andrey_gluschuk/lab_2/task_1/5.py:13]: Be sure to fix the "No newline at EOF" warning in future.

Comment (sen-den) [reply labs/andrey_gluschuk/lab_2/task_1/5.py:13]: Same to all other files.

File: labs/andrey_gluschuk/lab_2/task_5/1.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+'''
```
Comment (sen-den) [inline labs/andrey_gluschuk/lab_2/task_5/1.py:1]: Be sure to use """ instead of ''' for docstrings.

**Review by sen-den — APPROVED** [review]
Approved. Be sure to check warnings commented in the review next time.

File: labs/andrey_gluschuk/lab_2/task_1/5.py (hunk 1 above)
Comment (andxzrey) [reply labs/andrey_gluschuk/lab_2/task_1/5.py:13]: In the final version, when I defended the lab, I fixed it. but I uploaded not the latest version of the code to the github I was too lazy to fix it again, thank you 😂

**Review by Denchik34 — APPROVED** [review]
ok

**Pull request merged**